src/simsopt/geo/curve.py

```diff
@@ -653,6 +738,17 @@ def num_dofs(self):
         This function returns the number of dofs associated to the curve.
         """
         return self.curve.num_dofs()
+    
+    def centroid(self):
```

Comment:
hm, since JaxCurve is a Curve, doesn't `centroid` already exist for this class and is added in the PR?